addons/hw_drivers/iot_handlers/drivers/PrinterDriver_W.py

```diff
@@ -70,9 +71,11 @@ def print_report(self, data):
         stdout_log_level = logging.DEBUG
         try:
             ghostscript.Ghostscript(*args, stdout=stdout_buf, stderr=stderr_buf)
+            self.send_status(status="success")
         except Exception:
             _logger.exception("Error while printing report, ghostscript args: %s, error buffer: %s", args, stderr_buf.getvalue())
             stdout_log_level = logging.ERROR # some stdout value might contains relevant error information
+            self.send_status(status="error", message="Printing with Ghostscript failed")
```

Comment:
I'd say if this message would be transferred to Odoo db the user won't know what Ghostcript is so it would be nice to make it more generic